Doc/c-api/refcounting.rst

```diff
@@ -97,6 +97,39 @@ objects.
    :c:func:`Py_DECREF`, and the same warning applies.
 
 
+.. c:function:: PyObject* _Py_Borrow(PyObject *o)
+
+   :term:`Borrow a reference <borrowed reference>` to an object: decrement the
+   object reference count and return the object.
+
+   The object must not be ``NULL``; if you aren't sure that it isn't
+   ``NULL``, use :c:func:`_Py_XBorrow`.
+
+   This function can be used as an expression to update an old C extension to
+   newer C API functions which return a :term:`strong reference`. For example,
+   replace ``frame->f_code`` with ``_Py_Borrow(PyFrame_GetCode(frame))``.
+
+   If the object can be destroyed before the last usage of the borrowed
```

Comment:
Instead of calling `Py_INCREF` or `Py_NewRef`, I would recommend simply removing `_Py_Borrow` in the case where a strong reference is needed.

Reply:
Done